lib/node_modules/@stdlib/stats/base/dnanstdevch/manifest.json

```
@@ -27,55 +28,79 @@
   "confs": [
     {
       "task": "build",
+      "wasm": false,
       "src": [
-        "./src/dnanstdevch.c"
+        "./src/main.c"
       ],
       "include": [
         "./include"
       ],
-      "libraries": [
-        "-lm"
-      ],
+      "libraries": [],
       "libpath": [],
       "dependencies": [
+        "@stdlib/blas/base/shared",
+        "@stdlib/strided/base/stride2offset",
         "@stdlib/napi/export",
         "@stdlib/napi/argv",
         "@stdlib/napi/argv-int64",
         "@stdlib/napi/argv-double",
         "@stdlib/napi/argv-strided-float64array",
         "@stdlib/stats/base/dnanvariancech",
+        "@stdlib/math/base/special/sqrt",
         "@stdlib/napi/create-double"
       ]
     },
     {
       "task": "benchmark",
+      "wasm": false,
       "src": [
-        "./src/dnanstdevch.c"
+        "./src/main.c"
       ],
       "include": [
         "./include"
       ],
-      "libraries": [
-        "-lm"
-      ],
+      "libraries": [],
       "libpath": [],
       "dependencies": [
+        "@stdlib/blas/base/shared",
+        "@stdlib/strided/base/stride2offset",
+        "@stdlib/math/base/special/sqrt",
         "@stdlib/stats/base/dnanvariancech"
       ]
     },
     {
       "task": "examples",
+      "wasm": false,
       "src": [
-        "./src/dnanstdevch.c"
+        "./src/main.c"
       ],
       "include": [
         "./include"
       ],
-      "libraries": [
-        "-lm"
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/blas/base/shared",
+        "@stdlib/strided/base/stride2offset",
+        "@stdlib/math/base/special/sqrt",
+        "@stdlib/stats/base/dnanvariancech"
+      ]
+    },
+    {
+      "task": "",
```

```suggestion
      "task": "build",
```

You need to set this to the default task.